consensus/state_processing/src/per_block_processing/verify_bls_to_execution_change.rs

```diff
@@ -37,10 +37,9 @@ pub fn verify_bls_to_execution_change<T: EthSpec>(
         Invalid::NonBlsWithdrawalCredentials
     );
 
+    // Re-hashing the pubkey isn't necessary during block replay, so we may want to skip that in
+    // future.
     let pubkey_hash = hash(address_change.from_bls_pubkey.as_serialized());
-
-    // FIXME: Should this check be put inside the verify_signatures.is_true() condition?
-    //        I believe that's used for fuzzing so this is a Mehdi question..
```

Comment:
This had puzzled me for a while, but the answer is a very clear _no_. The pubkey hash is not checked as part of signature verification, so it _should not_ be grouped with it. Otherwise it would be skipped when using bulk signature verification for the block (most of the time).

Reply:
sounds good to me!